tests/jobs/ahm_analysis/test_integration.py

```diff
@@ -0,0 +1,102 @@
+import os
+import shutil
+import pytest
+import pandas as pd
+import semeio.workflows.ahm_analysis.ahmanalysis as ahmanalysis
+
+
+from res.enkf import EnKFMain, ResConfig
+from res.enkf.export import (
+    SummaryObservationCollector,
+    GenDataObservationCollector,
+)
+from ecl.grid import EclGridGenerator
+
+
+@pytest.mark.usefixtures("setup_tmpdir")
+def test_ahmanalysis_run(test_data_root):
+    """test data_set with only scalar parameters"""
+    test_data_dir = os.path.join(test_data_root, "snake_oil")
+
+    shutil.copytree(test_data_dir, "test_data")
+    os.chdir(os.path.join("test_data"))
+
+    res_config = ResConfig("snake_oil.ert")
+    res_config.convertToCReference(None)
+    ert = EnKFMain(res_config)
+
+    ahmanalysis.AhmAnalysisJob(ert).run(prior_name="default_0")
+
+    # assert that this returns/generates a KS csv file
+    output_ks = os.path.join(
+        "storage", "snake_oil", "reports", "default_0", "AhmAnalysisJob", "ks.csv"
+    )
+    assert os.path.isfile(output_ks)
+    ks_df = pd.read_csv(output_ks)
+    assert not ks_df.empty
+    output_activeobs = output_ks.replace("ks.csv", "active_obs_info.csv")
+    assert os.path.isfile(output_activeobs)
+    output_misfitobs = output_ks.replace("ks.csv", "misfit_obs_info.csv")
+    assert os.path.isfile(output_misfitobs)
+    output_prior = output_ks.replace("ks.csv", "prior.csv")
+    assert os.path.isfile(output_prior)
```

Comment:
Instead of manipulating, one could:

```
from pathlib import Path
outdir = "storage/snake_oil/reports/default_0/AhmAnalysisJob"
assert (outdir + "ks.csv").is_file()
...
```

Reply:
done